src/JuMP.jl

```diff
@@ -518,7 +519,7 @@ Base.copy(c::SDPConstraint, new_model::Model) =
 # ConstraintRef
 # Reference to a constraint for retrieving solution info
 immutable ConstraintRef{T<:JuMPConstraint}
-    m::Model
+    m::AbstractModel
```

Comment:
I was going to do that initially, but its a more clunky change. Under what scenario would it be bad though?